src/TestUtils/src/UITest.Appium/HelperExtensions.cs

```diff
@@ -17,7 +17,17 @@ public static class HelperExtensions
 		/// https://github.com/dotnet/maui/issues/19754
 		/// </summary>
 		/// <param name="app">Represents the main gateway to interact with an app.</param>
-		/// <param name="element">Target Element</param>
+		/// <param name="element">Target Element.</param>
+		public static void Tap(this IApp app, string element)
+		{
+			app.FindElement(element).Click();
+		}
```

Comment:
```suggestion
			=> Click(app, element);
```

Since this code is the same (for now) should we just refer it to that?